tbxforms/fields.py

```diff
@@ -52,21 +57,39 @@ def __init__(self, **kwargs):
                 label=_("Day"),
                 error_messages={"incomplete": _("Enter the day of the month")},
                 validators=[
-                    RegexValidator(r"^[0-9]+$", _("Enter a valid date"))
+                    RegexValidator(r"^[0-9]+$", _("Enter a valid date")),
```

Comment:
Looks like there's inconsistency with our error messages with some ending in full stops and others not (not as a result of this MR - looks like this has been an issue for a while).

Can we opt for ending with full stops, please?

Reply:
Added in 2fdcc2ae86fe279f9b523fdebafd5f9c44604c85